cli/viam/main.go

```diff
@@ -174,21 +178,25 @@ func main() {
 							if err != nil {
 								return err
 							}
-							for _, org := range orgs {
-								fmt.Fprintf(c.App.Writer, "%s (id: %s)\n", org.Name, org.Id)
+							for i, org := range orgs {
+								if i == 0 {
+									fmt.Fprintf(c.App.Writer, "Organizations for %q:\n", client.Config().Auth.User.Email)
+								}
```

Comment:
[nit/opt] why is this in the for loop with an `i==0` check rather than just being in front of the loop

Reply:
Putting this print statement at the top of loop will print:
```
Organizations for "benji.rewis@viam.com":
```
if there are no orgs associated with my user. I thought just printing nothing was better, but let me know if you disagree.